src/print_event_cb.c

```diff
@@ -296,11 +302,11 @@ fwrite_out_lf_fld_sep(FILE *f)
 void
 init_printing_events(void)
 {
-	static char str[] = "_----------------_----------------_";
+	static char str[] = "_------------------_------------------_";
+	char *c;
 
-	str[0]  = Args.separator;
-	str[17] = Args.separator;
-	str[34] = Args.separator;
+	while (c = strchr(str, '_'))
+		*c = Args.separator;
```

Comment:
Wait, what?
Modifying a string literal?
That is blasphemy!
Also undefined behavior.
Also, compiler can ask the linker to place these string literals in the rodata segment, which is going to be mapped as read-only memory pages - resulting in segfault.

I guess something like this would make me able to sleep at night:
```
static const char str_template[] = "_------------------_------------------_";
static char str_store[sizeof(str_template)];
strcpy(str_store, str_template);
```